### ISSUE TITLE
Remove history and make the back button work as intented

### ISSUE THREAD

**Issue by HuubSmit** [fancyapps/fancybox#1516]
Hello,

I have found an issue with the new fancybox3. The back button is not working as intended, as it moves through the pictures instead of going back a page.
Now I saw that there is a possibility to remove the hash, but that is not the solution, because the history isn't fixed by using that option. If I press back it still moves through the pictures, although now not anymore, because the url is not changed. So I press back and still be on the same page.

**Comment by HuubSmit** [comment]
Why did you close it? Do you have a solution?

**Comment by fancyapps** [comment]
I even do not know what to say, your description makes no sense. I understand that English is not your native language, but, please, try to combine words so that they make sense.

Also, pressing back button does close the modal.

**Comment by HuubSmit** [comment]
Let me try again then.

Pressing the back button is flipping through the pictures instead of going back pages, like a back button should supposed to work. I thought the issue was in the hash, as fancybox adds #picturename to it so you can actually hotlink to the actual picture in fancybox. But if I use the command to remove the hash, the history from the browser is still not working as intended, because it still adds the fancybox "pages".

**Comment by fancyapps** [comment]
Sorry, still no idea where is the issue. 

1) open http://fancyapps.com/fancybox/3/
2) click on any image
3) navigate through the gallery
4) click back button - gallery closes as expected.

**Comment by HuubSmit** [comment]
Sorry, I think the cache on my site gave problems. If I use the command to remove the hash it's working as intented. The backbutton goes to the previous page.

I'm sorry for the inconvenience.

**Comment by fancyapps** [comment]
Well, it will never work perfectly, because it is not possible to bind on "pressing back button" event. And you can manipulate with history entries, but you can not remove them.

**Comment by HuubSmit** [comment]
It now works perfectly though. I can scroll through the pictures but the history is now not affected by fancybox3, that was the issue I wanted to resolve!
